src/opentimelineio/clip.h

```diff
@@ -9,10 +9,18 @@ namespace opentimelineio { namespace OPENTIMELINEIO_VERSION {
 class Clip : public Item
 {
 public:
+    struct MediaRepresentation
```

Comment:
Naming check: everyone happy with MediaRepresentation? Any other suggestions?

Reply:
Also, I followed the model of Schema and Marker::Color which use a struct of strings rather than an enum with conversion functions between enum and string. I assume this is used because it makes the de/serialization easier.  If there's a better approach, let me know.